google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/JsonToProtoMessage.java

```diff
@@ -25,13 +25,11 @@
 import com.google.protobuf.Descriptors.Descriptor;
```

Comment:
Thanks for improving the code!
Could you provide a list of things refactored in the PR description?

Reply:
Updated